modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -205,18 +194,12 @@ class Configuration(object):
 
     @property
     def logger_format(self):
-        """The logger format.
-
-        The logger_formatter will be updated when sets logger_format.
-
-        :param value: The format string.
-        :type: str
-        """
```

Comment:
Why are we removing this comment?
It provides useful info about the type of this property and how to use it

Reply:
This is a getter not setter so the signature is wrong.

Reply:
Please add these docstrings back in.
When one uses help to find out how to use the methods of a class, it only gives back the docstring in the getter per:
https://stackoverflow.com/questions/16025462/what-is-the-right-way-to-put-a-docstring-on-python-property
Also having the getter describe the type returned by the getter his helpful to users